README.md

```diff
@@ -1,70 +1,67 @@
+
+ ## Developer IDE
+
+  * Visual studio code(https://code.visualstudio.com/download)
+
+  ### How to run sample using visual studio code
  
- * Open the solution file `BlazorWebAssembly.sln` in Visual studio. 
-
- * Open the EmbedProperties.cs file under the BlazorWebAssembly.Shared project.
-
- * Please change the following properties in the `EmbedProperties.cs` file per your Bold BI Server.
-
-<meta charset="utf-8"/>
-<table>
-  <tbody>
-    <tr>
-        <td align="left">UserEmail</td>
-        <td align="left">UserEmail of the Admin in your Bold BI, which will be used to get the dashboard list.</td>
-    </tr>
-    <tr>
-        <td align="left">EmbedSecret</td>
-        <td align="left">Get your EmbedSecret key from the Embed tab by enabling the `Enable embed authentication` on the Administration page https://help.boldbi.com/embedded-bi/site-administration/embed-settings/.</td>
-    </tr>
-  </tbody>
-</table>
-
-* Change the following properties in the script tag of the `index.html` file under the BlazorWebAssembly.client project in the following location, wwwroot/index.html per your Bold BI Server.
-
-<meta charset="utf-8"/>
-<table>
-  <tbody>
-    <tr>
-        <td align="left">RootUrl</td>
-        <td align="left">Dashboard Server URL (Eg: http://localhost:5000/bi, http://demo.boldbi.com/bi).</td>
-    </tr>
-    <tr>
-        <td align="left">SiteIdentifier</td>
-        <td align="left">For the Bold BI Enterprise edition, it should be like `site/site1.` For Bold BI Cloud, it should be an empty string.</td>
-    </tr>
-    <tr>
-        <td align="left">Environment</td>
-        <td align="left">Your Bold BI application environment. (If Cloud, you should use `cloud,` if Enterprise, you should use `enterprise`).</td>
-    </tr>
-    <tr>
-        <td align="left">dashboardId</td>
-        <td align="left">Provide the dashboard id of the dashboard you want to embed in view or edit mode. Ignore this property to create a new dashboard.</td>
-    </tr>
-  </tbody>
-</table>
-
-Please refer to the [help documentation](https://help.boldbi.com/embedded-bi/javascript-based/samples/v3.3.40-or-later/blazor-with-javascript/#how-to-run-the-blazor-webassembly-sample) to learn how to run the sample.
-
-## Online Demos
+  1. Open the [Blazor WebAssembly](https://github.com/boldbi/blazor-webassembly-sample/tree/master/BlazorWebAssembly) sample in Visual Studio Code.
+
+  2. Open the terminal and navigate to the `Server project` directory using the cd command(e.g., cd C:\BlazorProject\Server).
+   
+  3. To run the application, use the command `dotnet watch run` in the terminal. After executing the command, the application will automatically launch in the default browser. You can access it at the specified port number (e.g., http://localhost:5154/).
```

Comment:
Would application automatically launch in browser?

Reply:
yes